nextcloud-startup-script.sh

```diff
@@ -376,7 +376,8 @@ clear
 whiptail --title "Extra configurations" --checklist --separate-output "Choose what you want to configure\nSelect by pressing the spacebar" "$WT_HEIGHT" "$WT_WIDTH" 4 \
 "Security" "(Add extra security based on this http://goo.gl/gEJHi7)" OFF \
 "Static IP" "(Set static IP in Ubuntu with netplan.io)" OFF \
-"Automatic updates" "(Automatically update your server every week on Sundays)" OFF 2>results
+"Automatic updates" "(Automatically update your server every week on Sundays)" OFF \
+"CookieLifetime" "(Configure after what time every Nextcloud user gets logged out in the Browser)" OFF 2>results
```

Comment:
Does it fit here good? (below Security, Static IP and Automatic updates.)

Reply:
Yes, that's a better place for it.

Reply:
okay 👍